docs/root/version_history/current.rst

```diff
@@ -142,6 +142,7 @@ New Features
   useful in support systems such as CI, CD, etc. The
   :ref:`schema validator check tool <install_tools_schema_validator_check_tool>` has been added
   to the tools image.
+* udp_proxy: added :ref:`matcher <envoy_v3_api_field_extensions.filters.udp.udp_proxy.v3.UdpProxyConfig.matcher>` to support matching and routing to different clusters.
```

Comment:
Please add a note in deprecated below for the deprecated field.

Reply:
Ok, added.